mountpoint-s3/src/fs.rs

```diff
@@ -642,16 +643,15 @@ where
     }
 
     #[allow(clippy::too_many_arguments)] // We don't get to choose this interface
-    pub async fn read<R: ReadReplier>(
```

Comment:
can delete the `ReadReplier` trait now?

Reply:
true! not sure why it didn't warn about it being unused, dropping it now